core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -2016,6 +2018,58 @@ public void testSetPath()
                 .hasMessage("line 1:10: mismatched input '<EOF>'. Expecting: <identifier>");
     }
 
+    @Test
+    public void testSetTimeZone()
+    {
+        assertThat(statement("SET TIME ZONE LOCAL"))
+                .isEqualTo(
+                        new SetTimeZone(
+                                new NodeLocation(1, 1),
+                                Optional.empty()));
+        assertThat(statement("SET TIME ZONE 'America/Los_Angeles'"))
+                .isEqualTo(
+                        new SetTimeZone(
+                                new NodeLocation(1, 1),
+                                Optional.of(new StringLiteral(
+                                        new NodeLocation(1, 15),
+                                        "America/Los_Angeles"))));
+        assertThat(statement("SET TIME ZONE '-08:00'"))
+                .isEqualTo(
+                        new SetTimeZone(
+                                new NodeLocation(1, 1),
+                                Optional.of(new StringLiteral(
+                                        new NodeLocation(1, 15),
+                                        "-08:00"))));
+        assertThat(statement("SET TIME ZONE INTERVAL '10' HOUR"))
+                .isEqualTo(
+                        new SetTimeZone(
+                                new NodeLocation(1, 1),
+                                Optional.of(new IntervalLiteral(
+                                        new NodeLocation(1, 15),
+                                        "10", Sign.POSITIVE, IntervalField.HOUR, Optional.empty()))));
```

Comment:
put all arguments on one line

Reply:
Nope. It is a left-over. Thanks for bringing this up.